model/src/main/java/com/basistech/rosette/apimodel/recordsimilarity/records/RecordFieldType.java

```diff
@@ -16,16 +16,10 @@
 
 package com.basistech.rosette.apimodel.recordsimilarity.records;
 
-import com.fasterxml.jackson.annotation.JsonValue;
-import lombok.AllArgsConstructor;
-import lombok.Getter;
+public final class RecordFieldType {
```

Comment:
Out of curiosity, why change to a class rather than an enum?

Reply:
@anjacd RecordFieldType was originally an enum of name, date, and address for the three "types" of fields we allowed in records for record similarity. Now we allow any String given in to pass through as the value of the type for a RecordSimilarityFieldInfo object, so an enum no longer makes sense. The RecordFieldType class now exists so we can reference the String variables globally

Reply:
@gcalianese I guess I'm not understanding how a class makes more sense than an enum here, if all it's doing is holding the same three field types as the enum and the switch statement in RecordSimilarityDeserializerUtilities is interpreting anything beyond that set as an unknown field. From what I can see, the actual change to functionality is just that when the string isn't one of the name/date/address set, we treat it as an unknown field instead of erroring, which seems like it would have worked just as well with the original enum.

Reply:
@anjacd I think I see what you mean now. I suppose having a class with static String variables has only a slight advantage in that when you call, for example, RecordFieldType.RNI_NAME you get the String "rni_name" instead of having to call RecordFieldType.RNI_NAME.toString() to get the String value of the enum which we need to compare to the String type of a RecordSimilarityFieldInfo.

Reply:
@gcalianese Good point; my main concern was consistency with the RecordFieldType enum in rosette-server, but looking at it again they weren't ever identical to begin with so I'm not worried about it. Frankly, the effort of changing every instance back to the enum would be way more trouble than something this nitpicky is worth either way. LGTM